### PR TITLE
Simply modify the Kubectl logs information

### PR DESCRIPTION
**Pull request by navist2020**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Make help information easier to read
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ var (
 		kubectl logs nginx --all-containers=true
 
 		# Return snapshot logs from all containers in pods defined by label app=nginx
-		kubectl logs -lapp=nginx --all-containers=true
+		kubectl logs -l app=nginx --all-containers=true
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go:61]: I'd like to see this being consistent across all commands, have you checked all the examples if they are similar?
/lgtm cancel

Comment (navist2020) [reply staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go:61]: Thanks for your comments.I took a look for all help information about Kubectl.This seems not to exist in others.

**Comment by navist2020** [comment]
Hi @soltysh. This problem seems to not arise from other help information. Be there any that needs to be changed?
If you think there is, please let me know and I will solve it. Thanks! : )

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**